packages/@lwc/features/src/__tests__/flags.spec.ts

```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
-const { default: pluginTester } = require('babel-plugin-tester');
+import pluginTester from 'babel-plugin-tester';
 const plugin = require('../babel-plugin');
```

Comment:
Mixing `import` and `require` is a bit strange. Is there a reason not to `import plugin from '../babel-plugin'` and `export` in `babel-plugin`?

Reply:
Something to do with the Jest compiler. One only worked as an `import`, the other only worked as a `require`. I can add a comment.

Reply:
Nevermind, looks like I'm wrong. I can't remember why I added the `import`, but we don't need it. Reverted it.

Reply:
Nope, I was wrong. Fixed it in a more "elegant" way (see [this article](https://medium.com/@muravitskiy.mail/cannot-redeclare-block-scoped-variable-varname-how-to-fix-b1c3d9cc8206)) using `export {}`, and added a comment explaining what it does.